engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/register-client.html

```diff
@@ -0,0 +1,13 @@
+<li>
+  <p translate>Install client package:</p>
+
+  <p translate>To report package & errata information:</p>
+  <pre><code>yum -y install katello-host-tools</code></pre>
+
+  <p translate>To optionally report tracer information:</p>
+  <pre><code>yum -y install katello-host-tools-tracer</code></pre>
+
+  <p translate>For remote actions and reporting package & errata information:</p>
+  <pre><code>yum -y install katello-agent</code></pre>
+  <p translate>Learn more about these packages in the <a href="https://theforeman.org/plugins/katello/">Documentation</a> </p>
+</li>
```

Comment:
As per the issue report, do we also want to add:
```
If you want to use client tools such as katello-agent or puppet, the satellite-tools repository will need to be enabled

subscription-manager repos --enable=rhel-\*-satellite-tools-\*-rpms
```

Reply:
The satellite-tools repo is not really relevant for upstream, this change makes it possible to override/modify the registration steps.

The instructions to enable repo can then be easily added in downstream branding.